documentation/contributing/future_projects/index.markdown

```diff
@@ -4,14 +4,35 @@ comments: true
 date: 2017-4-9 02:13:26+00:00
 layout: page
 slug: code
-title: Code Sprints
+title: Future Projects
 ---
 
 # Potential Code Sprints and Future Projects
 
 <img src="/assets/logo/moveit_logo-black.png" width="300"/>
 
-This page lists potential projects that would greatly benefit the MoveIt project. This is by no means an exhaustive list, but is meant to serve as a discussion starting point for code sprints and other efforts. Please contact [PickNik Robotics](https://picknik.ai/connect/) for further information or sponsorship opportunities.
+This page lists potential projects that would greatly benefit the MoveIt project. This is by no means an exhaustive list, but is meant to serve as a discussion starting point for code sprints and other efforts.
+
+If you would like to contribute to these efforts, have a look at the related Github issues and ask for clarifications on [Github](https://github.com/ros-planning/moveit/issues) or [Discord](https://discord.gg/vqwEXwYU3N).
+
+Feel free to contact [PickNik Robotics](https://picknik.ai/connect/) for further information or sponsorship opportunities.
+
+
+# Table of Contents
+1. [TrajOpt Integration](#trajopt-integration)
+2. [Scene Graph Support](#scene-graph)
+3. [Benchmark Suite](#benchmark-suite)
+4. [MoveIt 2 Tutorials](#moveit-2-tutorials)
+5. [Port MoveIt Task Constructor to MoveIt 2](#mtc-in-moveit-2)
+6. [ros_control integration](#ros-control-integration)
+7. [Improve MoveIt Grasps](#moveit-grasps)
+8. [Improve Cartesian Planner](#cartesian-planner)
+9. [Warehouse Support](#warehouse-support)
+10.[MoveIt-OMPL Planning Interface](#ompl-interface)
+11.[Mobile Base Integration](#mobile-base-integration)
```

Comment:
```suggestion
10. [MoveIt-OMPL Planning Interface](#ompl-interface)
11. [Mobile Base Integration](#mobile-base-integration)
```

Reply:
Sorry for the weird/repeat comment flow. I'm still learning my way around github PR reviews...